lab_5_scrapper/scrapper.py

```diff
@@ -204,6 +310,10 @@ def prepare_environment(base_path: Union[pathlib.Path, str]) -> None:
     Args:
         base_path (Union[pathlib.Path, str]): Path where articles stores
     """
+    if not os.path.exists(base_path):
+        os.makedirs(base_path)
+    for file in os.listdir(base_path):
+        os.remove(os.path.join(base_path, file))
```

Comment:
do i need to remove directory as well (after removing all files) and then create a new one (i was a bit misled by documentations i guess it says this way)? because i found a way to remove only empty directory

Reply:
you need to do full cleanup, including directory removal

your base_path is Path type, you may find useful methods for this type that can help you remove directory

Reply:
did it